### ISSUE TITLE
Meeting Notes: April 18, 2022

### ISSUE THREAD

**Issue by MarsBarLee** [isabela-pf/internship#33]
[Draft](https://www.notion.so/Best-Practices-for-Accessible-Real-Time-Collaboration-Draft-ee043e54be364707a763e8c8cd2303e9)

## Backing up claim of remote of work
- "work began remotely and hence there was an increase in demands of RTC. "
- maybe add a link/citation

### Thinking of headers
- Current RTC accessibility options are insufficient
- Need for better RTC accessibility options
- Insufficient support of current RTC accessibility options

### People often want to use their own tools and workflows
- sometimes people prefer own tools, but want to integrate into RTC
- For example, with N interview, she likes using Mac Gestures, but cannot use them in Google Docs = frustrating
- RTC tool should allow for options
- Can follow WCAG guidelines for '[rubust](https://www.w3.org/TR/WCAG21/#robust)'
- "Content must be robust enough that it can be interpreted by by a wide variety of user agents, including assistive technologies."

## User Diagram
- Highlight what feels bad =  the 👎  point
  - 'Screen-reader reads out too much minor details, feels overwhelming'
- Where to write the full information? Recommendations? 
- Can write 

## Factor: Screen-readers reading out too many minor details instead of most important details
- when screen-readers read out too much, repeated information, poor user experience
- eg Google Doc comments
- screen-reader reading all HTML elements. Possibility-Developer add ARIA label to identify most important information that screen-reaader reads
## How to represent useful information from our non-collaborative test volunteers?
user journey diagram - visual, helpful, summarize data we collected such as with NVDA and VoiceOver

interview - turn to user journey diagram? or a different diagram? or text as 'case study: using Google Docs with X'? 
- anonymize concerns, don't want to put identifiable information

OR 'Key Takeaways for non-screen-reader accessibiltity'- talks about mac gestures and customization?

**Comment by isabela-pf** [comment]
Today: Review draft document with questions about feedback

- Links as proof, if it takes too long to find things

- Mars comment on is everything really accessible out there
- Following “who, what, where, when, why” so then what should go there?
- Mars: heading comment was about making it a little more specific and fitting with the theme elsewhere. Reflecting that things are a little more complicated.
- Are you referring to RTC tools, everything on the internet, etc.? Making it clear is helpful.

- "Internal vs. external tools" term clarificatiojn
- Internal tools meaning Google support for Google tools. Example: the Google-made screenreader can't be used on Google Docs (Isabela: I think they've changed support to be only on ChromeBooks?). And Swast mentioned there weren't tutorial resources, either, so even if things are working it's not well documented.
- Mars: we probably need better language or to be more specific (and just use more words). That likely won't be clear to someone reading this document with no other background.
- There doesn't seem to be a standard term for this.
- This seems related to [WCAG 2.1's Robust section](https://www.w3.org/TR/WCAG21/#robust)

- Question: user stories don't have all the info we found. How should we integrate them?
- Could be an annotation on the image
- Could be writing after the image with "Key takeaways" or "More details"

**Comment by MarsBarLee** [comment]
![image](https://user-images.githubusercontent.com/46167686/163825286-2015d131-d6d8-401b-bdbd-9af52e0ec5c5.png)

**Comment by MarsBarLee** [comment]
![image](https://user-images.githubusercontent.com/46167686/163825541-6216b36d-2caa-490b-b23e-e4c756e1fbfb.png)

**Comment by MarsBarLee** [comment]
You can just write text first, then see if you can visually present information, such as with tables

Don't fill table with data we don't have- that's guessing (well, an education guess but no based on our research data)

**Comment by isabela-pf** [comment]
I'm thinking something like this might be better suited for the research

| Disability (or category if that helps) | User feedback | Our recommendation  |
--- | --- | ---|
| Cognitive | "When I can't use gesture navigation, that makes my experience much worse." | Prioritize support for multiple info input types |

**Comment by MarsBarLee** [comment]
Mars likes Isabela's table 'User Feedback' section
- does not need to fit into linear journey, beginning to end like a user journey diagram
- can be difficult to rate 👍 or 👎 if not our own experiences

Yeehaw! Not all feedback or experiences are linear!
Love that
good take, Mars

**Comment by isabela-pf** [comment]
| Category  | User Feedback | Our recommendation |
| --------- | ---------------- | --------------------- |
| Cognitive | Try to zone it out, wait until everyone is done and read                    | Always give option to the user to decide the ammount of information to be shown |
| Temporary Disability          |    Accessibility should be considered from the very begining. Other socio-technological bariers can also be counted in here.                                                                         |Accessibility starts from inclusion.                                                                                 |
| Cognitive | "When I can't use gesture navigation, that makes my experience much worse." | Prioritize support for multiple info input types                                |
